pkg/api/customization/monitor/handler_utils.go

```diff
@@ -92,6 +90,7 @@ func newProjectGraphInputParser(input v3.QueryGraphInput) *projectGraphInputPars
 
 type projectGraphInputParser struct {
 	Input       *v3.QueryGraphInput
+	ProjectID   string
```

Comment:
What's the point we store project id, project name and cluster name? I think we have enough information stored here with project name and cluster name.

Reply:
@aiwantaozi , can you reply?

Reply:
We need projectID to get the project graph namespace, projectID include clusterName and projectName, so I remove projectName but keep projectID